snuba/subscriptions/store.py

```diff
@@ -26,10 +26,10 @@ def delete(self, key: UUID) -> None:
         pass
 
     @abc.abstractmethod
-    def all(self) -> Collection[Tuple[UUID, SubscriptionData]]:
+    def all(self) -> Iterable[Tuple[UUID, SubscriptionData]]:
```

Comment:
We don't depend on this being sized anywhere so I made this return type a bit more limited — nothing specific really motivated this change here.